config/crd/bases/ingress.pomerium.io_pomerium.yaml

```diff
@@ -36,6 +36,11 @@ spec:
           spec:
             description: PomeriumSpec defines Pomerium-specific configuration parameters.
             properties:
+              accessLogFields:
+                description: AccessLogFields sets the access fields to log.
+                items:
+                  type: string
```

Comment:
In CRD, we try to enumerate known values, so that k8s api server would be able to perform validation and prevent invalid config propagated to Pomerium.